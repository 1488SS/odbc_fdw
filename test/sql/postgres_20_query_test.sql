SELECT * FROM postgres_test_table;
SELECT * FROM query_postgres_test_table;
SELECT * FROM existent_table_in_schema_public;
SELECT * FROM test_table_in_schema;
SELECT * FROM ODBCTablesList('postgres_fdw', 1);