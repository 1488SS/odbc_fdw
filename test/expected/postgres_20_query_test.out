SELECT * FROM postgres_test_table;
 id | varchar_example | text_example | integer_example | numeric_example |    timestamp_example     | boolean_example 
----+-----------------+--------------+-----------------+-----------------+--------------------------+-----------------
  1 | example         | example      |             100 |       10.120000 | Fri Jan 01 00:00:00 2016 | t
(1 row)

SELECT * FROM query_postgres_test_table;
 count 
-------
     1
(1 row)

SELECT * FROM existent_table_in_schema_public;
ERROR:  Executing ODBC query
SELECT * FROM test_table_in_schema;
 id |  data   
----+---------
  1 | example
(1 row)

